app/assets/stylesheets/views/_homepage.scss

```diff
@@ -11,31 +11,60 @@ body.homepage {
   }
 }
 
-#homepage-promo-banner {
```

Comment:
I'm not sure why this diff isn't just a red block removing everything within `#homepage-promo-banner`.